.github/workflows/test.yml

```diff
@@ -15,34 +15,38 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
-      - name: Set up environment
-        run: ./.bin/set-up-globals.sh
       - name: Install Dependencies
         run: composer install
-      - name: Lint
-        run: composer lint
       - name: Unit Tests
         run: composer test
-      - name: Validate Terminus exists
-        run: terminus --version
   terminus-3:
     needs: unit
     runs-on: ubuntu-latest
     container:
       image: quay.io/pantheon-public/terminus-plugin-test:4.x-php8.0
       options: --user root
     name: Terminus 3 Functional Tests
+    env:
+      TERMINUS_SITE: terminus-addons-installer-plugin
+      TERMINUS_TOKEN: ${{ secrets.TERMINUS_TOKEN }}
+      BUILD_NUM: ${{ github.run_number }}
     steps:
       - name: Checkout
         uses: actions/checkout@v2
       - name: Set up environment
         run: ./.bin/set-up-globals.sh
-      - name: Validate Terminus exists
-        run: terminus --version
       - name: Install plugin
         run: terminus self:plugin:install ${GITHUB_WORKSPACE}
+      - name: Install SSH key
+        uses: webfactory/ssh-agent@v0.7.0
```

Comment:
Something I read about today and can't decide if I want to go down this route— pin this to a specific commit hash, as the tag could be compromised and tweet out your secrets.

Reply:
We could also do `webfactory/ssh-agent@master` but that seems sketchy. I would worry that pinning to a hash would be harder to maintain, but I guess not any harder than updating the release number (which, by the by, probably could/should be `0.8.0`, too...)

Reply:
Yeah it boils down to "do you/we trust this action with my secrets"? a branch or tag could be changed to nefarious code, but a specific git hash would not.

Reply:
I'm just paranoid enough to try it...